eZ/Publish/Core/Repository/ObjectStateService.php

```diff
@@ -224,7 +224,7 @@ public function updateObjectStateGroup(APIObjectStateGroup $objectStateGroup, Ob
             throw $e;
         }
 
-        return $this->buildDomainObjectStateGroupObject($spiObjectStateGroup);
+        return $this->buildDomainObjectStateGroupObject($spiObjectStateGroup, $objectStateGroup->prioritizedLanguages);
```

Comment:
Please add `@property-read string[] $prioritizedLanguages` to `\eZ\Publish\Core\Repository\Values\ObjectState\ObjectStateGroup`